install_config/install/advanced_install.adoc

```diff
@@ -1375,16 +1418,19 @@ ifdef::openshift-origin[]
 endif::[]
 ----
 
-If for any reason the installation fails, before re-running the installer, see
+. If for any reason the installation fails, before re-running the installer, see
 xref:installer-known-issues[Known Issues] to check for any specific
 instructions or workarounds.
 
```

Comment:
Yeah, it wasn't previously. There are two bits at the end of the install steps, basically "if it fails, see Known Issues" and "after it completes, verify the install." I can just keep them as non-numbered paragraphs that come after the procedure. I was entertaining making them all numbered since this section is going from a previously non-numbered single "step" to now multiple steps, if only temporarily.